src/core/node/crypto.js

```diff
@@ -9,6 +9,6 @@
 import { createHash } from "crypto";
 import { normalize8 } from "./bufutil.js";
 
-export function md5(b) {
-  return createHash("md5").update(normalize8(b)).digest("hex");
+export function sha256(b) {
+  return createHash("sha256").update(normalize8(b)).digest("hex");
```

Comment:
`md5` is used in a non cryptographic capacity. Merely for file integrity checks, like crc32.

These `md5` signatures are generated by [`trie.js`](https://github.com/serverless-dns/trie/blob/01336af6a0c4c962f474fcfe4ebf3bfcb83dcb35/src/trie.js#L995-L999)

Reply:
Should we switch from md5 to xxhash or crc32 as they as used as non cryptographic hashes and provide better performance and efficiency?

Reply:
> Should we switch from md5 to xxhash or crc32 as they as used as non cryptographic hashes and provide better performance and efficiency?

The rdns app expects `md5` so our hands are tied since older version of apps expect it (unless we want to break older versions). Besides, error detection + correction isn't really needed, just the checksum is fine, for which `md5` is well-suited.